src/pages/news/news-form/news-form.js

```diff
@@ -187,31 +139,27 @@ const NewsForm = ({ id, editMode }) => {
     dispatch(push(config.routes.pathToNews));
   };
 
-  if (loading) {
-    return <LoadingBar />;
-  }
-
   return (
-    <div className={styles.formContainer}>
-      <form onSubmit={formik.handleSubmit}>
+    <div>
```

Comment:
Why do you need another div here if you already set it in `src/pages/news/news-add/news-add.js `

Reply:
because form and back-button
![image](https://user-images.githubusercontent.com/46137635/104103940-a8d0ad00-52ad-11eb-9d39-a876d0c0c612.png)